src/knot_types.h

```diff
@@ -128,7 +128,7 @@
 #define KNOT_VALUE_TYPE_BOOL				0x03
 #define KNOT_VALUE_TYPE_RAW				0x04
 #define KNOT_VALUE_TYPE_MIN				KNOT_VALUE_TYPE_INT
-#define KNOT_VALUE_TYPE_MAX				KNOT_VALUE_TYPE_RAW
+#define KNOT_VALUE_TYPE_MAX				(KNOT_VALUE_TYPE_RAW+1)
```

Comment:
MAX and MIN can be used as constants to manipulate arrays. So, MIN shall point to the first element and MAX should point to the size of an array of elements (last element +1).